PS8lite.Rmd

```diff
@@ -180,30 +175,84 @@ MiscVal + MoSold + YrSold" %>%
 
 ***
 
+# Fitting a `randomForest` via `caret`
+
+
+## Setup cross-validation
 
+We're going use `caret` package functionality to
 
-# Fit a Random Forest model
+1. Define cross-validation settings. We are going to use cross-validation to generate model error estimates (when using fitted models to make predictions on new independent test data)
+1. Setup a "search grid" of "tuning" parameters to find the "optimal value." By "optimize" we mean pick the value of the "tuning" parameter that yields the lowest cross-validated error estimate.
 
-Fit one Random Forest model `model_rf` using any approach you want.
+The latter is like how in the:
+
+1. CART problem set you searched over a range of $\alpha$ values
+1. LASSO problem set you searched over a range of $\lambda$ values
+
+The "tuning" parameter we're going to "optimize" over is the number of randomly chosen variables we use at each split in our CART trees: `mtry`. We're going to search over values 2 through 5 (the maximum number of predictor variables we have access to). 
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.